packages/ai/core/tool/mcp/mcp-sse-transport.ts

```diff
@@ -37,6 +39,7 @@ export class SSEClientTransport implements MCPTransport {
           const response = await fetch(this.url.href, {
             headers: {
               Accept: 'text/event-stream',
```

Comment:
q: thoughts on spreading custom headers first and then setting `Accept` to avoid overriding?